### DISCUSSION TITLE
Beta "fiber" in FCC Metals

### DISCUSSION THREAD

**Discussion by bnmcbride** [mtex-toolbox/mtex#725] [Ask Anything]
This post is to try and summarize questions posed about the beta "fiber" in FCC metals. I think this will be quite useful for multiple users because intensity variation along the beta "fiber" is commonly reported in literature for FCC metals. This goes off other discussions listed in:

https://groups.google.com/g/mtexmail/c/pU0QG4D3ylw/m/UEDFgzVjAgAJ
https://groups.google.com/g/mtexmail/c/xelkIBWhm7I/m/_vo0ln1FCgAJ

I think the confusion in working with the beta "fiber" is that it is not a true fiber in the sense that it does NOT describe a fixed set of orientations in Euler space. Instead, the term "fiber" is used more loosely to describe texture components that fall between the Copper, S and Brass components in Euler space. It looks like other users have defined this as a sum of two "fibers": one between Copper and S and the other between Brass and S. I understand MTEX was not designed to handle this definition of a "fiber".  

My question is
1) Can texture intensity be plotted as a function of phi2 along this beta "fiber"? I see this as creating a list of all Euler space data points on a "fiber", and then only plotting the texture intensity against phi2 values. The phi2 should range from 45 (Copper) to 90 degrees (Brass).

When I try to do this using plot(odf,betaFiber), I get abscissa values that range from 0-35 degrees. I'm not sure what these represent, but I believe the issue is the definition of the beta "fiber".

**Comment by ralfHielscher** [comment]
In MTEX if you define the beta fibre as `f = fibre.beta(cs)` you get the line between the orientations `(112)[1-10]` and `(11 11 8)[-4 -4 11]`.
Accordingly, if you plot an ODF along the fiber `f` the range of 0 to 35 degrees describe the distance from `(112)[1-10]` on the way to `(11 11 8)[-4 -4 11]`. 

Note, that this is not the full beta fiber, but only a short section. You get the full beta fiber with
```matlab
f = fibre.beta(cs,'full')
```
Ralf.
